### DISCUSSION TITLE
Wrong path paths in .css (result of export Next.js)

### DISCUSSION THREAD

**Discussion by viktor-mozdolevskyi** [vercel/next.js#34888] [Help]
### What version of Next.js are you using?

12.0.2

### What version of Node.js are you using?

v14.15.1

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

next export

### Describe the Bug

I faced with next issue on my own project and on the test project (pure create-next-app). As the result of the next export, I receive wrong paths in CSS files, for example:
```
background: url(/images/oval_21.webp)
```
but should be
```
background: url(../../../images/oval_21.webp)
```
The same thing happens with fonts. Also, I just to note I am using 'next-optimized-images' for handling images in export and here my config setup:
```
module.exports = withPlugins([
  [optimizedImages, {
    optimizeImages: false,
  }],
  {
    assetPrefix: isDeployMode ? '.' : '',
    reactStrictMode: true,
    experimental: { esmExternals: true },
    images: {
      domains: ["localhost"],
    },
  }
]);
```

This CSS rule used in the Sass file:
```
#cq-landing {
  line-height: 1.5;
  background-color: #fff;
  font-family: 'SF Pro Display';
  font-display: swap;
  background: url(/images/oval_21.webp) top 0 center no-repeat;
  ...
}
```
And everything works fine with `next dev` and `next start`, issue reproduces only after export in static files.

### Expected Behavior

Correct paths in css files.

### To Reproduce

Add image on the background via background-mage: url(), and export static HTML via next export.

**Comment by olliethomas1992** [comment]
I am experiencing a similar issue. Did you ever find a solution @viktor-mozdolevskyi ?
